src/vcpu.rs

```diff
@@ -236,7 +213,11 @@ impl Aarch64VCpu {
     }
 }
 
-core::arch::global_asm!(include_str!("trap.S"));
+core::arch::global_asm!(
+    include_str!("trap.S"),
```

Comment:
> Do not use "trap", aarch64 or arm I think use exception is best!

if so, rename this `trap.S` into `exception.S` and move this inline assembly into `exception.rs`

Reply:
ok